etterlattemaler/src/main/kotlin/no/nav/pensjon/etterlatte/maler/Slatemodell.kt

```diff
@@ -27,6 +27,7 @@ data class InnerElement(
 enum class ElementType(@JsonValue val value: String) {
     HEADING_TWO("heading-two"),
     HEADING_THREE("heading-three"),
+    HEADING_FOUR("heading-four"),
```

Comment:
Sjekka no, og etterlatte må oppdatere litt på si side for å få brukt heading-four, ja. Men all den tid det er dei som har ønska seg funksjonaliteten, tenkjer eg at det må vera heilt greitt.